include/unifex/let_with_stop_source.hpp

```diff
@@ -123,7 +123,7 @@ class _stop_source_sender<SuccessorFactory>::type {
         (requires same_as<remove_cvref_t<Self>, type>)
     friend auto tag_invoke(tag_t<unifex::connect>, Self&& self, Receiver&& r)
         noexcept(
-            std::is_nothrow_constructible_v<SuccessorFactory, member_t<Self, SuccessorFactory>> &&
+            std::is_nothrow_invocable_v<SuccessorFactory, inplace_stop_source&> &&
```

Comment:
Sorry to nit, but I think this needs to be `is_nothrow_invocable_v<member_t<Self, SuccessorFactory>, inplace_stop_source&>` to match the value-category of the template-parameter passed to the operation-state.